root/forms/DialogResult.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * @flow strict
+ * Copyright (C) 2024 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import escapeClosingTags from '../utility/escapeClosingTags.js';
+
+component DialogResult(
+  result: string,
+) {
```

Comment:
After the message of your first commit:

> One important utility of root/forms/dialog.tt which I did port to React is in
how it returns the created entity JSON in a global variable on a blank page
after the form is submitted

* What is the name of this _utility_?
* What is the name of this _global variable_?
* Should `root/forms/dialog.tt` have a comment as below?
  ```perl
  # Converted to React at root/forms/DialogResult.js`
  ```

Reply:
1. By "utility" I actually meant "use" (noun), will fix the wording. So the use is just what's described in the sentence.
2. The name of the global variable is `dialogResult`.
3. Yes, I'll add that comment and clarify the commit message too.